source/07-processing/11-restating-data.rst

```diff
@@ -0,0 +1,20 @@
+.. The COUNTER Code of Practice Release 5 © 2017-2023 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+Restating data
+--------------
+
+
+Retrospective Reporting of Errors
+"""""""""""""""""""""""""""""""""
+
+If errors are identified in COUNTER reports, the report provider must correct the errors within three months of their discovery. The report provider must alert the Project Director so that a notification may be included in the COUNTER Registry. Report providers should also directly inform their customers of the corrections.
+
+
+Reporting Usage When Journal Titles Change
+""""""""""""""""""""""""""""""""""""""""""
+
+When the title of a journal is changed but the identification code (ISSN or DOI) stays the same, report providers should consider the journal to be a single ‘Title’ for the purposes of COUNTER reporting. COUNTER Reports and Standard Views of the COUNTER Reports should be provided against the new title, with the original title being dropped from the reports.
+
+If a new DOI or ISSN is allocated to the journal when the title changes, Report providers should consider the journal to be two ‘Titles’ and provide usage information separately. Report providers should not report usage for the same period under both the old and new DOI or ISSN. Any report generated for the old DOI or ISSN should show zero usage from the month in which the new DOI or ISSN takes effect.
```

Comment:
The single quotes also should be removed in this section.

SHOULD instead of should? The "should not" probably should be "MUST NOT", because not following that rule would result in double reporting.